app/supplejack/extraction/execution.rb

```diff
@@ -25,9 +25,13 @@ def call
 
         @de.extract_and_save
 
+        # The throttle needs to be before the transformation jobs are enqueued
+        # otherwise the child processes finish before the extraction process
+        # and the enrichments are never queued.
```

Comment:
Yeah sure, I will update the comment as well.

The child processes are the transformation and load workers. If you queue them and then wait for the throttle they finish before the extraction finishes. Since the load worker is queueing the enrichments when it finishes it checks to see if the harvest has finished and because the extraction worker is sleeping it is still running. So nothing queues the enrichments.

Reply:
Iv updated the comment :)